packages/main/src/themes/PopupsCommon.css

```diff
@@ -1,7 +1,6 @@
 :host {
     display: none;
     position: fixed;
-    min-width: 6.25rem;
```

Comment:
I saw that you reverted the removed `min-width: 20px` for Dialog.js, however this one is still deleted. I checked and Popover.css does not specify min-width by itself. Therefore `PopupsCommon.js` is responsible for setting it. After this change, popovers will no longer have a minimum width, which might be interpreted as a breaking change. Is there a reason to remove it?

Reply:
If min-width is set to the "host element" and there is not enough space, the Popover looks broken:
<img width="157" alt="MinWidth" src="https://user-images.githubusercontent.com/9462671/143565543-c9e6cf20-c270-4afe-9760-16fd3d8c7f37.PNG">

If it is removed, the Popover looks like:
<img width="96" alt="NoMinWidth" src="https://user-images.githubusercontent.com/9462671/143565649-b6052484-c2e6-449b-ad1d-fabe6cf894b4.PNG">

If we want to have some minimum width, we can add it to the "popover-root":
<img width="96" alt="MinWidthInside" src="https://user-images.githubusercontent.com/9462671/143565707-56c3860f-15c0-45ca-8a9b-3ac3540f39de.PNG">

which doesn't look so bad and we can use this approach. What do you think?

Reply:
I think it makes sense

Reply:
min width is added to the popover shadow root